src/discord/events/message/index.ts

```diff
@@ -16,8 +16,9 @@ type HandlerFn = (message: Discord.Message, context: DiscordContext) => Promise<
 
 interface Command {
   handlerId: string;
-  trigger?: string;
+  trigger?: RegExp;
   description: string;
+  readTrigger: string;
```

Comment:
I was unsure what this was at first, I think this is clearer

```suggestion
  displayTrigger: string;
```